index.test-d.ts

```diff
@@ -70,7 +70,7 @@ expectAssignable<LambdaFastifyOptions>({
   serializeLambdaArguments: false,
   decorateRequest: true,
   decorationPropertyName: "myAWSstuff",
-  enforceBase64: (reply) => false,
+  enforceBase64: (response) => false,
```

Comment:
We could actually improve this by
```suggestion
  enforceBase64: (response) => {
    expectType<LightMyRequestResponse>(response)
    return false
  },
```

we would need to import expectType from tsd and LightMyRequestResponse from fastify